### ISSUE TITLE
Wording of invite message mentions Chatsecure

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-iOS#535]
Android ticket with more info: https://github.com/zom/Zom-Android/issues/428

<img width="927" alt="screenshot 2018-02-08 21 24 38" src="https://user-images.githubusercontent.com/1434597/36010367-a4f37d14-0d16-11e8-93d5-8746fd7f1884.png">

**Comment by cstiens** [comment]
This message shouldn't be displayed at all. @N-Pex are you able to reproduce this on your iOS devices? I believe it shows up anytime you are added to a group, regardless of whether iOS or Android created it.

**Comment by N-Pex** [comment]
@cstiens Hm, no, I can't reproduce it?!?

**Comment by N-Pex** [comment]
Actually, now I could, when I logged out then in again. It's related to MAM catchups, when we get the group invites we inserted them in the message stream...

@chrisballinger PR sent. Is it possible, like we talked about, to somehow restrict the initial mam fetch in onAuthenticated to only fetch history for one-to-one chats, not groups? In this case the problem was that the "joined" flag was still false because we got the MAM stuff "too early".
